tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -16,9 +16,85 @@
 Documentation  Test 1-02 - Docker Pull
 Resource  ../../resources/Util.robot
 Suite Setup  Conditional Install VIC Appliance To Test Server
-Suite Teardown  Cleanup VIC Appliance On Test Server
+Suite Teardown  Local Teardown
 Test Timeout  20 minutes
 
+*** Keywords ***
+Local Teardown
+    Run Keyword  Cleanup VIC Appliance On Test Server
+    Run Keyword  Cleanup VCH Bridge Network  %{BRIDGE_NETWORK_2}
```

Comment:
just move this to a test [Teardown] and keep the original suite teardown.  Then eliminate this whole keyword.